### PR TITLE
App/spoolman

### PR DESCRIPTION
**Pull request by Nazim-ua**
Add app Spoolman 
https://github.com/Donkie/Spoolman
Spoolman is a web service that helps you keep track of your filament spools and how they are being used.

## Summary by CodeRabbit

- **New Features**
    - Introduced a web service "Spoolman" for managing 3D printer filament spools, enabling users to track spool usage and automate weight adjustments during printing.
    - Added Docker Compose configuration for seamless deployment of the Spoolman service.
    - Provided a detailed description of Spoolman, showcasing its features, integration with printer software, HTTP API, web client, and community translation support.
    - Improved deployment process with Docker Compose configuration for the Spoolman service.

### CODE REVIEW

File: apps/spoolman/docker-compose.yml
Comment (steveiliop56) [inline apps/spoolman/docker-compose.yml:1]: Please add a container name to the docker compose file. (something like this: `container_name: spoolman`)

File: apps/spoolman/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    image: ghcr.io/donkie/spoolman:0.16.1
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data:/home/app/.local/share/spoolman 
+    ports:
+      - ${APP_PORT}:8000
+    environment:
+      - TZ=${TZ}
+      - PUID=1000
+      - PGID=1000
```
Comment (steveiliop56) [inline apps/spoolman/docker-compose.yml:13]: The app has issues running with 1000, 1000 

```usermod: no changes
User UID: 1000
User GID: 1000
Starting uvicorn...
INFO:     Started server process [1]
INFO:     Waiting for application startup.
spoolman.env               WARNING  Data directory is not writable, trying to fix it...
chown: /home/app/.local/share/spoolman: Operation not permitted
chown: /home/app/.local/share/spoolman: Operation not permitted
spoolman.env               ERROR    Data directory is not writable. Please run "sudo chown -R 1000:1000 /path/to/spoolman/datadir" on the host OS.
ERROR:    Traceback (most recent call last):
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 566, in __aenter__
    await self._router.startup()
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 654, in startup
    await handler()
  File "/home/app/spoolman/spoolman/main.py", line 70, in startup
    env.check_write_permissions()
  File "/home/app/spoolman/spoolman/env.py", line 369, in check_write_permissions
    sys.exit(1)
SystemExit: 1

ERROR:    Application startup failed. Exiting.
```

so please remove the completely so the app runs as root.

**Review by steveiliop56 — COMMENTED** [review]
Hello @Nazim-ua,

Very nice work and very good app! I left you 2 small comments to fix and then we can merge!

**Review by Nazim-ua — COMMENTED** [review]
Thank you.
I have corrected your comments.

**Review by Nazim-ua — COMMENTED** [review]
Thank you.
I have corrected your comments.

**Review by Nazim-ua — COMMENTED** [review]
Thank you.
I have corrected your comments.
Sorry, this is my first PR.

**Comment by steveiliop56** [comment]
No worries your docker compose and config was very nice. It's like you have added 50 apps lol. Will test and merge.

**Review by steveiliop56 — APPROVED** [review]

**Pull request merged**